### ISSUE TITLE
[BUG} overlay issues

### ISSUE THREAD

**Issue by Veund** [chrismaltby/gb-studio#116]
im using the dev build of gb studio on windows 7

when i use the overlay function after a display text, the text box is on top of the overlay at the top of the screen 
![bug](https://user-images.githubusercontent.com/8967752/57176001-1fd95f80-6e96-11e9-91f2-08d0f5801c10.png)
![bug2](https://user-images.githubusercontent.com/8967752/57176002-2071f600-6e96-11e9-8bad-42ff6cb628dd.png)

**Comment by RichardULZ** [comment]
Intended, It's a feature!
The Gameboy's Overlay layer is used to draw text without breaking the background.
Use Show overlay to clear/make the overlay layer completely white or black.
![image](https://user-images.githubusercontent.com/34431457/57177242-7eaed100-6eb6-11e9-8f28-a52c1816c52e.png)

**Comment by Veund** [comment]
yeah, i know how they work, I'm saying the display text is getting stuck on the overlay when you run an overlay after a display text as shown on my second photo with the text box at the top of the screen

**Comment by RichardULZ** [comment]
It would be wasteful to clear the overlay layer after every text box, and the show overlay function (Not Move overlay) already has the option to clear.
Your script of 
```
Hide overlay,
Text "MORTALS!!!",
Wait 0.5 sec
```
Does not produce the shown result.

**Comment by Veund** [comment]
I may have missed a show overlay

Add a show overlay after the wait

**Comment by RichardULZ** [comment]
For anyone else that encounters this.
**Show Overlay** and **Move Overlay** are different functions, Show Overlay will fill the buffer White or Black, with a position, set that to X0 Y18 to clear it while off screen, 
Move overlay will only move it, not clear. 
This is Important for later features / text box functionality, and working as intended, 

**TL;DR solution.**
Use **Show Overlay** (Black) (offscreen) before **Move Overlay** after using a text box.
